### DISCUSSION TITLE
What is the NodeJs version for Serenity 8.1.5?

### DISCUSSION THREAD

**Discussion by lechu90ns** [serenity-is/Serenity#7052] [Help] [answered]
Hey Guys, how to recognize/check what is the current node js version for the latest Serenity package (8.1.5)? I am not sure if I am using the right version. Watch mode is working but not recognizing changes on files.

Thanks!

**Comment by VictorTomaili** [comment, marked as answer]
https://serenity.is/docs/getting_started/README#prerequisites
